src/main/java/nl/hsac/fitnesse/util/iban/BEIban.java

```diff
@@ -0,0 +1,232 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+/**
+ * Generates a Belgium IBAN.
+ */
+
```

Comment:
Ah, so that is what a JavaDoc comment looks like. I was not aware it was so I will read up on it and adjust accordingly.